### ISSUE TITLE
Add Versions for requested Semantic Models of IRS API

### ISSUE THREAD

**Issue by jzbmw** [eclipse-tractusx/item-relationship-service#439]
**As** PO ,
**I want** to be compliant to the versioning initiative of CX ,
**so that** we will be downward compatible .

## Acceptance Criteria

- [ ] the API Documentation has been updated
- [ ] the Tests have been updated
- [ ] The API has been changed to accept only the urn's which are available
- [ ] Add valid example values to API docu ( replace "string")
- [ ] FQN is used to define the aspect name 
- [ ] For detecting version we use the major version under account. 
   - [ ] [2.?.? ](urn:samm:io.catenax.single_level_bom_as_built:2.0.0) Accepting any minor a patch version of this aspect model
- [ ] Update Tavern and Cucumber Tests 
- [ ] Update documentation arc42, admin manual, etc  

## Hints / Details
Defining the aspect 2.0.0 the minor and patch version is ignored. The version 2.0.0 accepts any version between 2.0.0 and 3.0.0 
- example Value for an array element urn:samm:io.catenax.single_level_bom_as_built:2.0.0
![Image](https://github.com/eclipse-tractusx/item-relationship-service/assets/48311334/80d8db72-32e3-45f4-beac-598b327a3629)

- it shall be possible, that two versions of the same aspect can exist in parallel

````
apects [
      urn:samm:io.catenax.single_level_bom_as_built:2.0.0
]
````

## Out of Scope

- ...

**Comment by dsmf** [comment]
Pull Request: https://github.com/catenax-ng/tx-item-relationship-service/pull/807

**Comment by jzbmw** [comment]
the failing tavern tests will be accepted and checked again with the update of the Testdata to version 1.7.0
Issue was that Semantic Hub was not delivering valid schema files for SLBAB 2.0.0

**Comment by mkanal** [comment]
> the failing tavern tests will be accepted and checked again with the update of the Testdata to version 1.7.0 Issue was that Semantic Hub was not delivering valid schema files for SLBAB 2.0.0
